src/spec-jest/index.spec.js

```diff
@@ -0,0 +1,25 @@
+import chai, { expect } from "chai";
```

Comment:
This file is redundant, because it does the same thing as `examples/4_jestMode/spec.js`. Could you remove it?

In fact you can remove the whole spec-jest directory